libraries/core_libs/network/include/network/tarcap/packets_handlers/dag_block_packet_handler.hpp

```diff
@@ -21,7 +21,7 @@ class DagBlockPacketHandler : public ExtSyncingPacketHandler {
   virtual ~DagBlockPacketHandler() = default;
 
   void sendBlock(dev::p2p::NodeID const &peer_id, DagBlock block, const SharedTransactions &trxs);
-  void onNewBlockReceived(DagBlock &&block);
+  void onNewBlockReceived(DagBlock &&block, const std::shared_ptr<TaraxaPeer> &peer);
```

Comment:
maybe it should be by default peer = nullptr

Reply:
Fixed